hever/sass/_config-child-theme-deep.scss

```diff
@@ -70,37 +70,37 @@ $config-global: (
 	/* Colors */
 	"color": (
 		"primary": (
-			"default": #1279BE,
-			"hover": #303030,
+			"default": var(--wp--preset--color--primary),
```

Comment:
It's worth looking into if this code will be repeated for every theme that we apply this method to or there will be any differences and if we can avoid that repetition.